### PR TITLE
Update to rails 6

### PR DESCRIPTION
**Pull request by jkeck**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -9,7 +9,8 @@
 module Earthworks
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.1
+    config.load_defaults 6.0
+    config.autoloader = :classic # Our helper overrides don't work under the classic loader
```
Comment (jkeck) [inline config/application.rb:13]: Was really surprised about this, but I can confirm that the zeitwerk code loader appears to break how we override helpers.  Not sure if this is specific to EarthWorks or if we'll need to keep an eye out for this in other projects.

Comment (mejackreed) [reply config/application.rb:13]: huh.. thanks for pointing that out. We are probably at a good place if we need to change anything up in GeoBlacklight for a major release.

Comment (jkeck) [reply config/application.rb:13]: Thanks for writing the test that failed on the upgrade!  😆

**Pull request merged**